src/commands/env/env-get.js

```diff
@@ -16,9 +19,21 @@ const envGet = async (name, options, command) => {
     return false
   }
 
-  const siteData = await api.getSite({ siteId })
+  const { siteInfo } = cachedConfig
+  let { env } = cachedConfig
+
+  if (siteInfo.use_envelope) {
+    env = await getEnvelopeEnv({ api, context, env, key: name, scope, siteInfo })
+  } else if (context !== 'dev' || scope !== 'any') {
+    error(
+      `To specify a context or scope, please run ${chalk.yellowBright(
+        'netlify open:admin',
+      )} and opt in to the new Environment Variables experience`,
```

Comment:
[Sand] I think when Melanie and I talked about this a while back, we thought it would be helpful to be more specific about what happens when you run the command and where to go:

```suggestion
      )} to open the Netlify UI and opt in to the new Environment Variables experience from Site settings`,
```

They won't see the env vars section if they don't enable it in Labs first. But I think it's ok to say Site settings here because they only need to enable it in Labs once and they'll have to go back to Site settings for every site they want to enable afterwards.   What do you think?

Reply:
Also, in the API docs, we lowercased `environment variables experience`, let's do that here to keep it consistent. 

```suggestion
      )} to open the Netlify UI and opt in to the new environment variables experience from Site settings`,
```

Reply:
Nice, I prefer mentioning Site settings over Labs too -- the feature might not always be in Labs, so we should avoid a temporary location b/c the user might be living on an old / out-of-date version of the CLI.